src/HtmlProcessor/CssToAttributeConverter.php

```diff
@@ -164,11 +165,12 @@ private function mapSimpleCssProperty(string $property, string $value, \DOMEleme
         }
 
         $mapping = $this->cssToHtmlMap[$property];
-        $nodesMatch = !isset($mapping['nodes']) || \in_array($node->nodeName, $mapping['nodes'], true);
-        $valuesMatch = !isset($mapping['values']) || \in_array($value, $mapping['values'], true);
+        $nodesMatch = !isset($mapping['nodes']) || \in_array($node->nodeName, (array)$mapping['nodes'], true);
+        $valuesMatch = !isset($mapping['values']) || \in_array($value, (array)$mapping['values'], true);
         $canBeMapped = $nodesMatch && $valuesMatch;
         if ($canBeMapped) {
-            $node->setAttribute($mapping['attribute'], $value);
+            $attributeMapping = \is_string($mapping['attribute']) ? $mapping['attribute'] : '';
+            $node->setAttribute($attributeMapping, $value);
```

Comment:
If we can be more explicit in the type hint for `$cssToHtmlMap`, the `is_string` test wouldn't be needed - ?

Reply:
Done.